### ISSUE TITLE
addStickyValidationError only after second submit

### ISSUE THREAD

**Issue by garenor** [aldeed/meteor-autoform#755]
Hi,

I love the idea of the sticky validation error in 5(.0.2), but I'm having an issue with it. I implemented it as follows:
````
AutoForm.addHooks('updateShipIdentificationForm', {
    after:{
        "method-update": function (error, result) {
            if (error) {
                this.addStickyValidationError("identification.imo", "notUnique");
            }
        }
    }
});
````
My method throws a dummy error message, and the sticky validation error is set. But, it is not visible on the screen until I submit my form for the second time.

My form is as follows:

````
{{#autoForm schema="shipIdentificationSchema" doc=ship id="updateShipIdentificationForm" type="method-update" meteormethod="updateShipIdentification"}}
{{> afQuickField name="identification.imo"}}
{{> afQuickField name="identification.mmsi"}}
{{/autoForm}}
````

Any ideas?

**Comment by aldeed** [comment]
hmm, in my testing, I think I was doing it in an `onError` hook, so you could try that instead. (It will also get called with the same error if there is one.)

I'm guessing that this is a bit of a bug in the current implementation, though. The sticky errors don't get added until the next validation run, so we should be triggering a form revalidation after submission ends if there are any sticky errors added.

**Comment by aldeed** [comment]
Calling `AutoForm.validateForm('updateShipIdentificationForm')` after your `addStickyValidationError` call should work as a workaround.

**Comment by garenor** [comment]
yep, came up with the same solution. Thanks!

**Comment by aldeed** [comment]
Fixed in 5.7.0
